DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:101 Selenium Manager binary found at: /opt/hostedtoolcache/Python/3.12.6/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:114 Executing process: /opt/hostedtoolcache/Python/3.12.6/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager --browser chrome --debug --language-binding python --output json
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Found chromedriver 127.0.6533.119 in PATH: /opt/hostedtoolcache/setup-chrome/chromedriver/127.0.6533.119/x64/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 chrome detected at /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Running command: /usr/bin/google-chrome --version
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Output: "Google Chrome 129.0.6668.58 "
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Detected browser: chrome 129.0.6668.58
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Discovering versions from https://googlechromelabs.github.io/chrome-for-testing/known-good-versions-with-downloads.json
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Required driver: chromedriver 129.0.6668.70
WARNING  selenium.webdriver.common.selenium_manager:selenium_manager.py:138 The chromedriver version (127.0.6533.119) detected in PATH at /opt/hostedtoolcache/setup-chrome/chromedriver/127.0.6533.119/x64/chromedriver might not be compatible with the detected chrome version (129.0.6668.58); currently, chromedriver 129.0.6668.70 is recommended for chrome 129.*, so it is advised to delete the driver in PATH and retry
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Driver path: /opt/hostedtoolcache/setup-chrome/chromedriver/127.0.6533.119/x64/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Browser path: /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.service:service.py:219 Started executable: `/opt/hostedtoolcache/setup-chrome/chromedriver/127.0.6533.119/x64/chromedriver` in a child process with pid: 2466 using 0 to output -3
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:305 POST http://localhost:46627/session {'capabilities': {'firstMatch': [{}], 'alwaysMatch': {'browserName': 'chrome', 'pageLoadStrategy': <PageLoadStrategy.normal: 'normal'>, 'browserVersion': None, 'goog:chromeOptions': {'extensions': [], 'binary': '/usr/bin/google-chrome', 'args': ['--headless=new', '--window-size=1920,1080']}}}}
DEBUG    urllib3.connectionpool:connectionpool.py:243 Starting new HTTP connection (1): localhost:46627
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://localhost:46627 "POST /session HTTP/11" 500 0
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:334 Remote response: status=500 | data={"value":{"error":"session not created","message":"session not created: This version of ChromeDriver only supports Chrome version 127\nCurrent browser version is 129.0.6668.58 with binary path /usr/bin/google-chrome","stacktrace":"#0 0x5601649f76ca \u003Cunknown>\n#1 0x5601646c8600 \u003Cunknown>\n#2 0x56016470713c \u003Cunknown>\n#3 0x5601647060b2 \u003Cunknown>\n#4 0x560164700f24 \u003Cunknown>\n#5 0x5601646fc2f8 \u003Cunknown>\n#6 0x560164746ce8 \u003Cunknown>\n#7 0x56016473a643 \u003Cunknown>\n#8 0x56016470ad31 \u003Cunknown>\n#9 0x56016470b79e \u003Cunknown>\n#10 0x5601649bf25b \u003Cunknown>\n#11 0x5601649c31f2 \u003Cunknown>\n#12 0x5601649ac615 \u003Cunknown>\n#13 0x5601649c3d82 \u003Cunknown>\n#14 0x56016499125f \u003Cunknown>\n#15 0x5601649e6e68 \u003Cunknown>\n#16 0x5601649e7040 \u003Cunknown>\n#17 0x5601649f649c \u003Cunknown>\n#18 0x7ffa37094ac3 \u003Cunknown>\n"}} | headers=HTTPHeaderDict({'Content-Length': '889', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:361 Finished Request
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:101 Selenium Manager binary found at: /opt/hostedtoolcache/Python/3.12.6/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:114 Executing process: /opt/hostedtoolcache/Python/3.12.6/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager --browser chrome --debug --language-binding python --output json
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Found chromedriver 127.0.6533.119 in PATH: /opt/hostedtoolcache/setup-chrome/chromedriver/127.0.6533.119/x64/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 chrome detected at /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Running command: /usr/bin/google-chrome --version
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Output: "Google Chrome 129.0.6668.58 "
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Detected browser: chrome 129.0.6668.58
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Required driver: chromedriver 129.0.6668.70
WARNING  selenium.webdriver.common.selenium_manager:selenium_manager.py:138 The chromedriver version (127.0.6533.119) detected in PATH at /opt/hostedtoolcache/setup-chrome/chromedriver/127.0.6533.119/x64/chromedriver might not be compatible with the detected chrome version (129.0.6668.58); currently, chromedriver 129.0.6668.70 is recommended for chrome 129.*, so it is advised to delete the driver in PATH and retry
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Driver path: /opt/hostedtoolcache/setup-chrome/chromedriver/127.0.6533.119/x64/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Browser path: /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.service:service.py:219 Started executable: `/opt/hostedtoolcache/setup-chrome/chromedriver/127.0.6533.119/x64/chromedriver` in a child process with pid: 2952 using 0 to output -3
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:305 POST http://localhost:56231/session {'capabilities': {'firstMatch': [{}], 'alwaysMatch': {'browserName': 'chrome', 'pageLoadStrategy': <PageLoadStrategy.normal: 'normal'>, 'browserVersion': None, 'goog:chromeOptions': {'extensions': [], 'binary': '/usr/bin/google-chrome', 'args': ['--headless=new', '--window-size=1920,1080']}}}}
DEBUG    urllib3.connectionpool:connectionpool.py:243 Starting new HTTP connection (1): localhost:56231
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://localhost:56231 "POST /session HTTP/11" 500 0
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:334 Remote response: status=500 | data={"value":{"error":"session not created","message":"session not created: This version of ChromeDriver only supports Chrome version 127\nCurrent browser version is 129.0.6668.58 with binary path /usr/bin/google-chrome","stacktrace":"#0 0x55a6e2f0d6ca \u003Cunknown>\n#1 0x55a6e2bde600 \u003Cunknown>\n#2 0x55a6e2c1d13c \u003Cunknown>\n#3 0x55a6e2c1c0b2 \u003Cunknown>\n#4 0x55a6e2c16f24 \u003Cunknown>\n#5 0x55a6e2c122f8 \u003Cunknown>\n#6 0x55a6e2c5cce8 \u003Cunknown>\n#7 0x55a6e2c50643 \u003Cunknown>\n#8 0x55a6e2c20d31 \u003Cunknown>\n#9 0x55a6e2c2179e \u003Cunknown>\n#10 0x55a6e2ed525b \u003Cunknown>\n#11 0x55a6e2ed91f2 \u003Cunknown>\n#12 0x55a6e2ec2615 \u003Cunknown>\n#13 0x55a6e2ed9d82 \u003Cunknown>\n#14 0x55a6e2ea725f \u003Cunknown>\n#15 0x55a6e2efce68 \u003Cunknown>\n#16 0x55a6e2efd040 \u003Cunknown>\n#17 0x55a6e2f0c49c \u003Cunknown>\n#18 0x7fd8d3294ac3 \u003Cunknown>\n"}} | headers=HTTPHeaderDict({'Content-Length': '889', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:361 Finished Request